packages/docs/getting-started/running-a-validator.md

```diff
@@ -135,6 +137,15 @@ export CELO_IMAGE=us.gcr.io/celo-testnet/celo-node:baklava
 export NETWORK_ID=1101
 ```
 
+For having the Proxy and the Validator reporting to the network EthStats service, you can export the following 2 environment variables:
```

Comment:
nit: "To have the Proxy and Validator report to the network EthStats service"...